### DISCUSSION TITLE
The size in the docker container

### DISCUSSION THREAD

**Discussion by edwardglenarvan** [tobychui/zoraxy#230] [General]
Thank you for the excellent program! It was only with her that I managed to set up a "secure login" to my site. Before that, it was impossible to configure with NGINX. The only thing I wanted to clarify about Zoraxy is that its size in my container is 2.85 Gb. While NGINX takes 142 Mb. Is this how it should be? Is it probably because of the GUI and libraries of the Goland language? I am grateful in advance for the answer.

**Comment by PassiveLemon** [comment]
We actually just got 2 PRs that reduce the size of the Docker image so once the next version is released, the containers will be much smaller (Less than 200 MB). They will still be larger than NGINX though because Zoraxy does run a WebUI.

**Reply by edwardglenarvan** [reply]
THX
